pkg/cmd/extension/manager.go

```diff
@@ -249,6 +250,64 @@ func (m *Manager) installDir() string {
 	return filepath.Join(m.dataDir(), "extensions")
 }
 
+func (m *Manager) Create(name string) error {
+	exe, err := m.lookPath("git")
+	if err != nil {
+		return err
+	}
+
+	err = os.Mkdir(name, 0755)
+	if err != nil {
+		return err
+	}
+
+	initCmd := m.newCommand(exe, "init", "-b", "main", "--quiet", name)
+	err = initCmd.Run()
+	if err != nil {
+		return err
+	}
+
+	fileTmpl := heredoc.Docf(`
+		#!/bin/bash
+		set -e
+		echo "Hello %[1]s"
+
+		# Examples to help get started:
+
+		# Determine if an executable is in the PATH
+		# if [ "$(which go)" = "" ]; then
+		#   echo "go not found in PATH"
```

Comment:
I would use `type` to check whether an executable is in PATH. It doesn't require a subshell and it's a shell builtin as opposed to a separate process. Furthermore, `which` can have subtle differences across platforms.

I would also check for things other than Go because I don't think it's realistic right now that people will be able to make Go extensions without us giving them any resources to do so with. It's much more likely that they will use a scripting language like Ruby or Python.

Finally, I would make sure that all error/warning output goes to stderr, if we're going to set good examples for people's scripts.
```suggestion
		# if ! type -p ruby >/dev/null; then
		#   echo "Ruby not found on the system" >&2
```